src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/content.test.ts

```diff
@@ -0,0 +1,131 @@
+import languageAssertions from '../../../../../../test/unit/utils/languageAssertions';
+import { FormContent, FormFields, FormInput, FormOptions, LanguageLookup } from '../../../../../app/form/Form';
+import { isFieldFilledIn, isTextAreaValid } from '../../../../../app/form/validation';
+import { CommonContent, generatePageContent } from '../../../../common/common.content';
+
+import { generateContent } from './content';
+
+jest.mock('../../../../../app/form/validation');
+
+const en = {
+  caption: 'Safety concerns',
+  title: 'Provide details of the previous abductions',
+  line1: 'Give a short description of the previous incidents of abduction.',
+  c1A_previousAbductionsShortDescHint: 'Include any previous attempts to threaten or abduct the children.',
+  c1A_policeOrInvestigatorInvolved: 'Were the police, private investigators or any other organisation involved?',
+  c1A_policeOrInvestigatorInvolvedHint: 'Including in the UK or overseas.',
+  one: 'Yes',
+  two: 'No',
+  otherDetails: 'Provide more details',
+  guidanceParentalChildHyperlink: 'https://www.gov.uk/government/collections/child-abduction',
```

Comment:
@niranjan-301 Let's remove these common links & use it from common files present in abduction.

Reply:
done